packages/plugin-credentials/src/credentials/common.js

```diff
@@ -215,10 +224,12 @@ export default {
   @oneFlight
   @waitForValue(`@`)
   refresh() {
+    if (this.isLoggingOut) {
+      return Promise.reject(new Error(`credentials: Cannot refresh while logging out`));
```

Comment:
`credentials: Credentials#refresh() is not available while Credentials#isLoggingOut is true`